### DISCUSSION TITLE
Share between subdomains

### DISCUSSION THREAD

**Discussion by costalino** [Blazored/LocalStorage#123] [General]
Hello all,

There is a way to share storage between subdomain (xxx.domain.com and yyy.domain.com)

Regards

Lino

**Comment by chrissainty** [comment]
Hi @costalino,

A quick Google shows up some... creative (read hacky) ways you might be able to achieve this but officially the answer is no. This is to make the storage more secure. Take GitHub as an example, when you publish a GitHub pages site, it runs from a subdomain of GitHub.io. If local storage allowed sharing between sub-domains, then any GitHub Pages site would be able to access any others local storage data.
